gui/ui/src/component/amount.rs

```diff
@@ -92,9 +91,7 @@ fn render_amount<'a, T: 'a>(amount: String, size: u16) -> Row<'a, T> {
 fn render_unconfirmed_amount<'a, T: 'a>(amount: String, size: u16) -> Row<'a, T> {
     let spacing = if size > P1_SIZE { 10 } else { 5 };
 
-    let row = Row::new()
-        .spacing(spacing)
-        .push(text(amount).size(size).style(color::GREY_3));
+    let row = Row::new().push(text(amount).size(size).style(color::GREY_3));
```

Comment:
This Row::new seems to be useless, please move text(amount) to the next Row::with_children block

Reply:
Good catch, thanks. I just made this change.